app/shared/shared/mapping/models.py

```diff
@@ -468,3 +470,38 @@ class Meta:
 
     def __str__(self) -> str:
         return str(self.id)
+
+
+class MappingTool(BaseModel):
+    """
+    Model for a Mapping Tool.
+
+    Attributes:
+        name (CharField): The name of the mapping tool.
+        version (CharField): The version of the mapping tool.
+    """
+
+    name = models.CharField(max_length=128)
+    version = models.CharField(max_length=64)
+
+    class Meta:
+        app_label = "mapping"
+        verbose_name = "Mapping Tool"
+        verbose_name_plural = "Mapping Tools"
+
+    def __str__(self):
+        return f"{self.name} v{self.version}"
+
+
+class MappingRecommendation(BaseModel):
+    # These two fields are required for a generic relation
+    content_type = models.ForeignKey(ContentType, on_delete=models.CASCADE)
+    object_id = models.PositiveIntegerField()
+    scan_report_value = GenericForeignKey("content_type", "object_id")
+
+    concept = models.ForeignKey(Concept, on_delete=models.CASCADE)
+    score = models.FloatField()
```

Comment:
In the first instance `score` should be `null=True, blank=True)`, as not everything will have a Score.